### DISCUSSION TITLE
Perfecto dedicated wdio service?

### DISCUSSION THREAD

**Discussion by rakesh747** [webdriverio/webdriverio#7949] [Q&A]
Is there a dedicated wdio perfecto service to run the tests on perfecto cloud devices such as services provided for Browserstack, Sauce Labs, and TestingBot?

**Comment by christian-bromann** [comment]
With a little Google foodoo I found https://www.npmjs.com/package/wdio-perfecto-service

**Reply by rakesh747** [reply]
Thank you for responding and after installing I'm getting the below error -->

npm ERR! In file included from /Users/gadirajr/Library/Caches/node-gyp/16.9.0/include/node/v8.h:30:
npm ERR! /Users/gadirajr/Library/Caches/node-gyp/16.9.0/include/node/v8-internal.h:489:38: error: no template named 'remove_cv_t' in namespace 'std'; did you mean 'remove_cv'?
npm ERR!             !std::is_same<Data, std::remove_cv_t<T>>::value>::Perform(data);
npm ERR!                                 ~~~~~^~~~~~~~~~~
npm ERR!                                      remove_cv
npm ERR! /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/type_traits:697:50: note: 'remove_cv' declared here
npm ERR! template <class _Tp> struct _LIBCPP_TEMPLATE_VIS remove_cv
npm ERR!                                                  ^
npm ERR! 1 error generated.
npm ERR! make: *** [Release/obj.target/integer/src/integer.o] Error 1
npm ERR! gyp ERR! build error 
npm ERR! gyp ERR! stack Error: `make` failed with exit code: 2
npm ERR! gyp ERR! stack     at ChildProcess.onExit (/Users/gadirajr/.nvm/versions/node/v16.9.0/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:194:23)
npm ERR! gyp ERR! stack     at ChildProcess.emit (node:events:394:28)
npm ERR! gyp ERR! stack     at Process.ChildProcess._handle.onexit (node:internal/child_process:290:12)
npm ERR! gyp ERR! System Darwin 19.6.0
npm ERR! gyp ERR! command "/Users/gadirajr/.nvm/versions/node/v16.9.0/bin/node" "/Users/gadirajr/.nvm/versions/node/v16.9.0/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
npm ERR! gyp ERR! cwd /Users/gadirajr/automation/wdio_perfecto/node_modules/integer
npm ERR! gyp ERR! node -v v16.9.0
npm ERR! gyp ERR! node-gyp -v v7.1.2
npm ERR! gyp ERR! not ok

**Reply by rakesh747** [reply]
these are my dependencies -->

    "dependencies": {
        "webdriverio": "^7.16.13"
    },
    "devDependencies": {
        "@babel/core": "^7.16.12",
        "@babel/preset-env": "^7.16.11",
        "@babel/register": "^7.16.9",
        "@wdio/cli": "^7.16.13",
        "@wdio/local-runner": "^7.16.13",
        "@wdio/mocha-framework": "^7.16.13",
        "@wdio/spec-reporter": "^7.16.13",
        "chromedriver": "^97.0.2",
        "wdio-chromedriver-service": "^7.2.6",
        "wdio-perfecto-service": "~0.1.0"
    },

**Reply by rakesh747** [reply]
@christian-bromann I got a response from the respective dev saying 'We no longer maintain wdio-perfecto-service project"
